001_sdk_L457/001_sdk_L457_safe.py

```
@@ -0,0 +1,504 @@
+import pickle
+import functools
+import argparse
+import importlib
+from pathlib import Path
+from xmlrpc.client import ServerProxy
+from typing import Dict, List, Tuple
+from enum import Enum
+
+try:
+    from liqi import LiqiProto, MsgType
+except:
```

Ruff (0.14.1)

Do not use bare `except`

(E722)